src/go/rpk/pkg/api/admin/api_broker.go

```diff
@@ -60,3 +60,10 @@ func (a *AdminAPI) EnableMaintenanceMode(node_id int) error {
 		fmt.Sprintf("%s/%d/maintenance", brokersEndpoint, node_id),
 		nil, nil)
 }
+
+// DisableMaintenanceMode disables maintenance mode for a node.
+func (a *AdminAPI) DisableMaintenanceMode(node_id int) error {
```

Comment:
Same comments as in https://github.com/redpanda-data/redpanda/pull/4255/commits/50df094475f9ea8fac307c0e6fc04ca08ba4aa4f